test_schema.sql

```diff
@@ -0,0 +1,16 @@
+-- name: create-users-table
```

Comment:
Yeah we can do that. Probably will be cleaner for tests. I'll add a few more tables and break out into a separate file.